### PR TITLE
Make Marathon port optional

### PR DESCRIPTION
**Pull request by ssk2**

### CODE REVIEW

File: dcos-cli/install.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ usage()
   echo "install.sh <shared_secret> <installation_path> <marathon_host> <marathon_port>"
```
Comment (jsancio) [inline dcos-cli/install.sh]: Should we change this to `
echo "install.sh <shared_secret> <installation_path> [<marathon_host> [<marathon_port>]]`? Maybe that is too hard to deal with?

File: dcos-cli/install.sh [hunk 2]
```diff
@@ -17,7 +17,7 @@ ARGS=( "$@" );
 SECRET=${ARGS[0]}
 VIRTUAL_ENV_PATH=$(python -c "import os; print(os.path.realpath('"${ARGS[1]}"'))")
 MARATHON_HOST=${ARGS[2]}
```
Comment (jsancio) [inline dcos-cli/install.sh]: should we default this to the dns entry to the marathon host which is also the mesos master?

Comment (ssk2) [inline dcos-cli/install.sh]: I think Marathon host is sufficient. We'll just point out in our instructions that users should pass in the DNS host for this value and let them work out what's actually happening if they choose to introspect the script.

File: dcos-cli/install.sh (hunk 1 above)
Comment (ConnorDoyle) [inline dcos-cli/install.sh]: @jsancio yeah, let's add some defaults here.  Do you want to take it on?

Comment (ssk2) [inline dcos-cli/install.sh]: Fixed!

File: dcos-cli/install.sh [hunk 3]
```diff
@@ -4,10 +4,10 @@ set -o errexit -o pipefail
 
 usage()
 { # Show usage information.
-  echo "install.sh <shared_secret> <installation_path> <marathon_host> <marathon_port>"
+  echo "install.sh <shared_secret> <installation_path> <marathon_host> [<marathon_port>]"
 }
 
-if [ "$#" -ne 4 ]; then
+if [ "$#" -ne 3 ]; then
```
Comment (jsancio) [inline dcos-cli/install.sh]: Actually, this should be `<` in `[` it is `-lt` I think.

File: dcos-cli/install.sh (hunk 2 above)
Comment (jsancio) [inline dcos-cli/install.sh]: You are right! Maybe not the DNS host but maybe the ELB which to them it will be the same host as the DCOS UI I think.

Comment (ssk2) [inline dcos-cli/install.sh]: That's the one, I've documented it!

File: dcos-cli/install.sh (hunk 3 above)
Comment (ssk2) [inline dcos-cli/install.sh]: Amended, good catch!

**Comment by jsancio** [comment]
Squash commits! Looks good to me!

**Pull request merged**